### PR TITLE
Adapt to changes in the API library

### PR DESCRIPTION
**Pull request by greschd**
Adapt to the change moving the file format definition to `enum_types.proto`:

Adds an optional argument `explicit_value_list` to the enum wrapper, which
specifies which of the wrapped values are exposed.

This is used to wrap the `enum_types_pb2.FileFormat` in different ways, exposing
only the file formats acceptable for a given use.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -38,7 +38,7 @@ numpy = ">=1.22"
 grpcio-health-checking = ">=1.43"
 packaging = ">=15.0"
 typing-extensions = ">=4.5.0"
-ansys-api-acp = "^0.1.dev9"
+ansys-api-acp = ">=0.2.0.dev0,<0.4.0"
```
Comment (roosre) [inline pyproject.toml:41]: Why 0.4 and not 0.3?

**Review by roosre — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (greschd) [inline pyproject.toml:41]:
```suggestion
ansys-api-acp = ">=0.2.0.dev0,<0.3.0"
```

Comment (greschd) [reply pyproject.toml]: Should have been 0.3, the change was unintentional.

**Comment by greschd** [comment]
Using a pinned API library version for now; otherwise we'd have to use the `--pre` flag when installing with `pip`.

Not sure why this worked previously, could be because we now fully specify the version as `0.2.0.dev0` (instead of `0.2.dev0`) in the `VERSION` file of the API library. Or something changed about how poetry packages the wheel or pip consumes it.

In any case, I think we can switch to proper releases for the API library with the next version bump.

**Pull request merged**